ccpp/data/GFS_typedefs.meta

```diff
@@ -5667,18 +5967,12 @@
   units = flag
   dimensions = ()
   type = logical
-[grav_settling]
-  standard_name = control_for_gravitational_settling_of_cloud_droplets
-  long_name = flag to activate gravitational setting of fog
-  units = flag
-  dimensions = ()
-  type = integer
 [bl_mynn_tkebudget]
   standard_name = control_for_tke_budget_output
```

Comment:
Since this changed from integer to logical, the standard name should change to `do_tke_budget_output`

Reply:
Not for now. Until the other bl_mynn_whatever options change type, we should keep the same naming convention to avoid user confusion. I'm unclear as to why the type changed, but maybe @joeolson42 can tell us.

Reply:
I kind of regret agreeing to change this to a logical variable. It was requested by some WRF developers who are leveraging this flag for other schemes too. So I am also taking "mynn" out of the name. In my attempt to maintain universalized code, I changed it to logical here in CCPP too.